src/sentry/static/sentry/app/views/groupDetails/shared/eventToolbar.jsx

```diff
@@ -82,14 +83,19 @@ const GroupEventToolbar = createReactClass({
   render() {
     const evt = this.props.event;
 
-    const {orgId, projectId} = this.props;
+    const {organization, orgId, projectId} = this.props;
     const groupId = this.props.group.id;
 
+    const hasSentry10 = new Set(organization.features).has('sentry10');
+    const baseIssuesPath = hasSentry10
+      ? `/organizations/${orgId}/issues/`
+      : `/${orgId}/${projectId}/issues/`;
+
     const eventNavNodes = [
       evt.previousEventID ? (
         <Link
           key="oldest"
-          to={`/${orgId}/${projectId}/issues/${groupId}/events/oldest/`}
```

Comment:
It looks like all of these are identical all the way though `${groupId}/events/` so if you wanted to cut down on some duplication you could move that stuff up to `baseIssuesPath`

Reply:
Thanks - updated